packages/graphback-core/src/db/dataMapper.ts

```diff
@@ -0,0 +1,55 @@
+import { ModelTableMapping } from './createModelTableMappings';
+
+export interface DataMapping {
+    id?: TableID
+    table?: string
+    data?: any
+    fieldMap?: any
+}
+
+export interface TableID {
+    name: string
+    value: any
+}
+
+export const mapDataToTable = (data: any, modelMap: ModelTableMapping): DataMapping => {
```

Comment:
Would `mapDTO` be a better name here?

Reply:
DTO stands for DataTransferObject and it will be confusing

Reply:
Name is ok